### ISSUE TITLE
Iterations on descriptions

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#135]
@emily-phet @ariel-phet @jessegreenberg
Below is a draft of a potential new scene summary for Balloons and Static Electricity. I drafted it a while ago and thought I better share it for comment.
It is longer than the previous one, but provides more context. It follows the Big to small concept. The navigation cue is much more playful and makes it clear a user should try to be in the Play Area.

Note that the bracketed phrases will have to be dynamically updated. 

When you have time please comment.
>__Scene Summary__
> This simulation contains a Play Area and a Control Panel.
> 
> The Play Area is a small room. It has [[a balloon, a sweater and a removable wall]].
> 
> The Balloon is in the [[middle]] of Play Area, [[evenly between sweater and wall]]. Sweater is at far left, wall is at far right.
> 
> Sweater [[and wall have many pairs of negative and positive charges, the balloon, just a few pairs]]. Charges can change as you play and learn about static electricity.
>  
> The Control Panel allows you to add a second balloon, remove the wall and reset your experiments.
> 
> Tab to play with Balloon. Tab again to check options in Control Panel. Shift-Tab to come back to Play Area and play with Balloon.

**Comment by jessegreenberg** [comment]
@terracoda looking good from my end, but I do have a couple of questions that we may wish to discuss here or at the next meeting.

- Is 'removable wall' in the play area description also a phrase that should be dynamically updated?

- For the phrase 'Tab to play with Balloon...' is there any reason to believe that this kind of cue would prevent users from using their normal strategies of navigation?

- A few weeks ago we began to talk about structuring this kind of content with additional html so that the user can easily find the content that they are looking for in certain descriptions.  I don't think we ever finished that conversation.  Did you envision something like that for this description?

**Comment by terracoda** [comment]
@jessegreenberg 

- Regarding your first point, yes any reference to the wall would have to be dynamic. I indicated "removable" just to ignite curiosity like the button does in the visual. However, it might not be necessary. There are other kinds of implicit scaffolds in the non-visual experience. The square brackets indicate changing content. I adjusted the second line.
- I certainly don't want to prevent users from using a different navigation strategy. I discussed this issue with her. I think this is something that we should ask a few screen reader users, and unfortunately, it is not something I directly asked my 12 participants. Hind sight is 20/20. I could have asked a lot more questions about the design :-)
-Yes, we need structure. I can provide structure - no problem! I would like to know how aria-live updates will work with a list of details rather than a paragraph.

**Comment by jessegreenberg** [comment]
> Regarding your first point, yes any reference to the wall would have to be dynamic. 

Sounds good!

> I think this is something that we should ask a few screen reader users...

Ok, this sounds great to me.  If we tell users to use 'tab' to navigate I was just wondering if they would avoid using other navigation strategies.

> Yes, we need structure. I can provide structure - no problem! 

Again, sounds good!  I wasn't sure where we landed on this in previous discussions.  I am also curious about how aria-live will behave with other structures and lists.

**Comment by terracoda** [comment]
@jessegreenberg The user may avoid using a different navigational strategy, but if that strategy doesn't work, they will likely try the Tab key if they know about it.. We know the Tab key will always work. Other things will work, too, but we are not sure that all screen reader navigation commands will work. I have a feeling most work.

**Comment by terracoda** [comment]
@jessegreenberg is it possible for you to implement this new scene summary text? @ariel-phet and @emily-phet haven't responded, so I think it is fine. It is better than what we have now :-) 

I have made some adjustment to the text and added a list structure with comments. I've decided to keep the navigation cue simple with just the one suggestion.

``` html
<h2>Scene Summary</h2>
<ul>
<!-- Room description -->
<li>This simulation contains a Play Area and a Control Panel. The Play Area is a small room. It has [[a balloon, a sweater and a removable wall]].</li>
<!-- The Balloon's position in relation to other objects -->
 <li>The Balloon is in the [[middle]] of Play Area, [[evenly between sweater and wall]]. Sweater is at far left[[, wall is at far right]].</li>
<!-- Charge Information -->
<li>Sweater [[and wall have many pairs of negative and positive charges, the balloon, just a few pairs]]. Charges can change as you play and learn about static electricity.</li>
<!-- Mini details of last release event (Maybe - just an idea - and not there on page load) -->
<!-- <li>On last release Balloon attracted to Sweater.</li> --> 
<!-- Control panel description (static) -->
<li>The Control Panel allows you to add and remove the wall, add and remove a second balloon, and reset Balloon charges and position.</li>
<!-- Navigation cue -->
<li>Tab to play with Balloon.</li>
</ul> ```

**Comment by ariel-phet** [comment]
@terracoda sorry did not see this before...but it no major comments for the moment. I get lots of github traffic, so if you want to make sure I see something please don't hesitate to give me an email nudge.

**Comment by terracoda** [comment]
@ariel-phet Thanks for the note! Will do!

**Comment by jessegreenberg** [comment]
Working on this today, I noticed that we will need one list item for each balloon.  @terracoda how would you like the list items to read when the green balloon is invisible?  I am imagining that the green balloon's list item could be invisible with the balloon, or the text could change to something like 'The green balloon is removed from play area."  I slightly prefer the latter.

**Comment by jessegreenberg** [comment]
Completed in the commit above, but I need to fully test to make sure that all possible dynamic descriptions are correct.  It is working nicely with NVDA, and I will also test with the screen reader prototype once it supports list items.

**Comment by jessegreenberg** [comment]
Closing this issue, list above works and we have new description content.
